homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -481,8 +484,7 @@ def native_value(self) -> int | None:
         ):
             return None
 
-        # Convert from g/m³ to mg/m³
-        return int(float(value) * 1000)
+        return round(value, 3)
```

Comment:
Should we use `_attr_suggested_display_precision = 3` instead of rounding here?

Reply:
this may lead to massive recorder writes, as the native value could have much more decimal places, so rounding to a sane value should be IMO fine :thinking:

Reply:
Ok, just wanted to confirm 🙂